packages/cozy-sharing/src/components/EditableSharingModal.jsx

```diff
@@ -33,7 +33,8 @@ export const EditableSharingModal = ({
           revokeSharingLink,
           hasSharedParent,
           hasSharedChild,
-          revokeSelf
+          revokeSelf,
+          getSharingType
```

Comment:
It seems like a smell that we need to have all this in the context ? Why is it necessary ?

Reply:
Cette histoire de passer le sharingType est remplacée par sharing par la suite

Mais aujourd'hui on passe l'ensemble des méthodes du sharing car on ne passait jamais le sharing en tant que tel. Donc on utilisait les méthodes du context pour récupérer les info en se basant sur le state. 

Maintenant que le sharing est passé, on va pouvoir commencer à faire des méthodes pour le manipuler sans dépendre du state et ce sera certainement mieux. 

Mais ce sera pour une autre fois